### PR TITLE
File for LORIS contributing guidelines

### PR DESCRIPTION
**Pull request by Jkat**

### CODE REVIEW

**Comment by driusan** [comment]
Shouldn't the guidelines have content?

**Comment by Jkat** [comment]
Indeed, I wanted to bring this up during the next pull request meeting to discuss the content. There are some docs floating around that we can pull parts from and incorporate it nicely here.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#LORIS Contributing Guidelines
+
+For every pull request:
+
+# Developing
+ * Don’t forget sql patch and Schema commits.  (e.g. New modules require LorisMenu and LorisMenuPermissions inserts)
```
Comment (rathisekaran) [inline CONTRIBUTING.md]: Clarify to say default schema

**Comment by driusan** [comment]
I don't like this content for the file.

If anything, as an outside person who stumbled across this repository it would make me less likely to  contribute. It reads like a list of rules written in point form without context for a potential external contributor (which is the target audience of a contributing.md file) who stumbles across it. Some of them don't even make sense (if you stumble across an open source project and want to contribute, what does "Test outside your sandbox if reasonable" mean? I'm a Loris developer, and I don't even understand what "* Document the changes in the Wiki" means. What changes am I documenting? Where are they going in the wiki and why?)

I think a good CONTRIBUTING.md file should make sense to someone who has no familiarity with Loris, and give encouragement and pointers to resources that exist for someone who wants to contribute but doesn't know where to start (ie pointers to the loris-dev list, a link to the developers workshop wiki page, suggestions that they can flesh out some of the documentation that they don't understand or run phpcs on some modules (and an explanation of how) if they don't know where to start, a link to the code review checklist so they understand how pull requests are evaluated, etc.)

**Comment by Jkat** [comment]
Thank you for the comments. I took all these contents from the code review checklist actually. I put that here as a starting point for discussion otherwise there would be nothing substantial to discuss initially. Maybe we should also review the code review checklist at the same time if there are points in there that don't make as much sense as they could. I also put a point in the meeting agenda earlier in hopes of getting rid of the existing google doc eventually, if possible, so that it is one less thing to have to go through.

**Pull request closed without merging**